packages/react-devtools-extensions/src/utils.js

```diff
@@ -1,10 +1,34 @@
 /* global chrome */
 
-import type {BrowserTheme} from 'react-devtools-shared/src/devtools/views/DevTools';
+import type { BrowserTheme } from 'react-devtools-shared/src/devtools/views/DevTools';
 
-export const IS_EDGE = navigator.userAgent.indexOf('Edg') >= 0;
-export const IS_FIREFOX = navigator.userAgent.indexOf('Firefox') >= 0;
-export const IS_CHROME = IS_EDGE === false && IS_FIREFOX === false;
+export let IS_EDGE;
+export let IS_FIREFOX;
+export let IS_CHROME;
+
+async function getBrowserData() {
+  if (navigator.userAgentData) {
+    navigator.userAgentData.brands.forEach(brand => {
+      switch (brand.brand.toLowerCase()) {
+        case 'google chrome':
+          IS_CHROME = true;
+          break;
+        case 'firefox':
+          IS_FIREFOX = true;
+          break;
+        case 'edge':
+          IS_EDGE = true;
+          break;
+      }
+    });
+} else {
+  IS_EDGE = navigator.userAgent.indexOf('Edg') >= 0;
+  IS_FIREFOX = navigator.userAgent.indexOf('Firefox') >= 0;
+  IS_CHROME = IS_EDGE === false && IS_FIREFOX === false;
```

Comment:
Please run `yarn prettier-all` to fix all linter issues

Reply:
For specific security reasons, I cannot pull local now and run terminal commands on the code. Last changes were directly through Github. If it is possible you can run yarn prettier-all and suggest changes I would be grateful. Thanks for your time Ruslan.